doc/changes/changes_6.0.0.md

```diff
@@ -1,6 +1,10 @@
 # Exasol Virtual Schema 6.0.0, released 2022-01-18
 
-Code name: Support native Exasol data types
+Code name: Support native Exasol data types (retracted)
```

Comment:
If you call the new one 6.0.1 there is no need for this. It will be deprecated anyway.